lib/resolvers/project.js

```diff
@@ -83,6 +90,12 @@ module.exports = ({ models }) => async ({ action, data, id, meta }, transaction)
       licenceNumber: project.licenceNumber || licenceNumber
     };
 
+    const lastAmendedDate = await getLastAmendedDate();
```

Comment:
I don't think this is needed. The date either doesn't need to be set (i.e. it's a first time application and so `grantedVersion` won't exist) or it's "now".